runtime/src/launch/slurm-srun/launch-slurm-srun.c

```diff
@@ -240,6 +243,21 @@ static char* chpl_launch_create_command(int argc, char* argv[],
     memStr = memEnv;
   }
 
+  localesPerNode = 1;
+  if (lpnEnv != NULL) {
+    localesPerNode = atoi(lpnEnv);
```

Comment:
I _think_ we can use `chpl_env_str_to_int` wrapper (or maybe can just use `chpl_env_rt_get_int`?)

Reply:
Changed it to use `chpl_env_rt_get_int`.